src/core/index.ts

```diff
@@ -438,7 +456,7 @@ export class ZarrArray {
     return new NestedArray<T>(buffer, this.chunks, this.dtype);
   }
 
-  private decodeChunk(chunkData: ValidStoreType) {
+  public decodeChunk(chunkData: ValidStoreType) {
```

Comment:
Did you mean to keep this public?

Reply:
removed for now (since not relevant to this PR), but I've been thinking a bit lately about whether we should make this method public. Hopefully we handle all cases of "types" of requests users will want to make with `get`, `getRaw`, and `getRawChunk` and thus no one should need to decodeChunk manually.